Makefile

```diff
@@ -20,3 +20,35 @@ install:
 .PHONY: changelog
 changelog:
 	@git-chglog --next-tag $(tag) $(tag)
+
+
+.PHONY: compile
```

Comment:
Can these changes be moved out to a separate PR please? While potentially useful, they are not related to the addition of the sort by feature.

Reply:
Sure thing, no worries :)